wire-ios-sync-engine/Tests/Source/Synchronization/ZMSyncStrategyTests.m

```diff
@@ -91,8 +91,10 @@ - (void)setUp
     ZMConversation *selfConversation = [ZMConversation insertNewObjectInManagedObjectContext:self.syncMOC];
     selfConversation.remoteIdentifier = self.userIdentifier;
     selfConversation.conversationType = ZMConversationTypeSelf;
-    
-    self.syncMOC.zm_lastNotificationID = [NSUUID UUID];
+
+    LastEventIDRepository *lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.userIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
same

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d